tchannel/zipkin/tracers.py

```diff
@@ -148,10 +153,18 @@ def submit_callback(f):
                     exc_info=f.exc_info()
                 )
 
+        span_host = self.parse_host_port()
```

Comment:
the zipkin trace hook is added usually before tchannel starts listening. Therefore the tchannel host port is not ready at __init__.